plugin/shell_plugin.go

```diff
@@ -0,0 +1,29 @@
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/alibaba/sealer/utils/ssh"
+)
+
+type Sheller struct {
+}
+
+func (s Sheller) Run(context Context, phase Phase) error {
+	//get cmdline content
```

Comment:
if phase != context.Plugin.Spec.On {
   return nil
}

if the phase not mach plugin `on` feild, ignore it.